docs/platforms/dotnet/common/metrics/index.mdx

```diff
@@ -93,3 +97,58 @@ using (SentrySdk.Metrics.StartTimer("bingo"))
     // Your code goes here
 }
 ```
+
+# OpenTelemetry Metrics
+
+The Sentry SDK for .NET also allows you to configure OpenTelemetry metrics that you would like to automatically capture and send to Sentry.
+
+For example, you could configure Sentry to capture all of the built-in metrics that it knows about:
+
+```csharp
```

Comment:
```suggestion
```csharp {13}
```
Not sure if we want to highlight line 13 here to call out how to capture all metrics

Reply:
Oh, that's awesome. I didn't know that was even possible 😁